prog/postgres/postgres_server_nexus.rb

```diff
@@ -295,6 +293,25 @@ def before_run
     nap 5
   end
 
+  label def configure_hugepages
```

Comment:
This is a way, to push so much to clover (also, use daemonizer2 and helper symbols if you are going to dothat)

One that may work just as well or better is to compute the shared memory size with every `systemctl restart` with a `ExecStartPre`. This is robust to slight fluctuations that crop up with updating extension versions or tweaking settings, otherwise you have to haul in this whole loop again for what is some basic arithmetic. The result is something like, "I'm doing some simple work and systemctl restart doesn't work and then I gotta haul in some semaphores to make this arithmetic work"

The downside is every restart is slightly slower for running the extra exec of Postgres emitting the hugepage number.